### PR TITLE
Raamit

### PR DESCRIPTION
**Pull request by auramo**
R̡̳̝̹̹͎͉ͫͭ͂A̗̞̪̥̹̫̎̌̎͑Á̤̰͌̍M̳̳̟̦̼̐ͤ̊ͯͅI͖ͯ̋̒ͮͮ́́Ṭ̩̟̐̊ͭ͌

### CODE REVIEW

File: resources/less/virkailija-banner.less [hunk 1]
```diff
@@ -36,88 +33,89 @@
...
+  visibility: hidden;
+  height: @banner-height;
+}
+
+.local-dev-logout {
+  position: absolute;
+  right: -36px;
+  top: 186px;
+  transform: rotate(90deg);
+  > a { text-decoration: none;
```
Comment (wolverian) [inline resources/less/virkailija-banner.less]: Weird indentation here

File: resources/less/virkailija-banner.less [hunk 2]
```diff
@@ -36,88 +33,89 @@
...
+        color: red;
+        color: red;
```
Comment (wolverian) [inline resources/less/virkailija-banner.less]: Identical lines

File: src/clj/ataru/virkailija/virkailija_routes.clj [hunk 3]
```diff
@@ -278,6 +280,32 @@
...
+     :headers (map-kv
+               (fn [header-kw header-value] [(name header-kw) header-value])
+               (:headers request)))))
+
+;; All these paths are required to be proxied by raamit when running locally
+;; in your dev-environment. They will get proxied to the correct test environment
+;; (e.g. itest/luokka or qa)
+(api/defroutes local-raami-routes
+  (api/undocumented
+   (api/GET "/virkailija-raamit/*" request;[* headers]
```
Comment (wolverian) [inline src/clj/ataru/virkailija/virkailija_routes.clj]: Commented code, remove?

File: resources/less/virkailija-banner.less (hunk 1 above)
Comment (auramo) [inline resources/less/virkailija-banner.less]: will fix

File: resources/less/virkailija-banner.less (hunk 2 above)
Comment (auramo) [inline resources/less/virkailija-banner.less]: will fix

File: src/clj/ataru/virkailija/virkailija_routes.clj (hunk 3 above)
Comment (auramo) [inline src/clj/ataru/virkailija/virkailija_routes.clj]: will fix

**Pull request merged**